### PR TITLE
FP-740: Create Form styles

### PR DESCRIPTION
**Pull request by gstvdesign**
# Ticket(s)

- [FP-740: Create Form styles](https://fourkitchens.clickup.com/t/86a2g7cxp)

## Purpose

* Create styles for hubspot forms

**This PR does the following:**

* create the form styles
* create necessary stories for input, selectfield, radio button, and checkbox
* export styles to be used in other apps
* create necessary icons for selectfield and checkbox

### Functional Testing

- [x] Checkout to this branch and run `npm install && npm run dev`
- [x] Look for the form components

Screenshot 2024-03-11 at 15 09 42
Screenshot 2024-03-11 at 15 09 50
Screenshot 2024-03-11 at 15 10 05
Screenshot 2024-03-11 at 15 10 20

### CODE REVIEW

**Comment by fkbender** [comment]
Task linked: [FP-740 Create Form styles](https://app.clickup.com/t/36718269/FP-740)

File: src/lib/assets/css/form.css
Comment (mcortes19) [inline src/lib/assets/css/form.css:1]: @gstvdesign could you please configure your text editor to always add an empty line at the end of the file?

File: src/lib/assets/css/form.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
+@tailwind base;
```
Comment (mcortes19) [inline src/lib/assets/css/form.css:1]: I think you don't need to add `@tailwind` imports again

**Review by mcortes19 — CHANGES_REQUESTED** [review]
@gstvdesign great work! 

I just found some issues need to be fixed

I linked this branch to my local version of Next.js to test out the sign-up form and noticed the following:

- We need to add space between each form field: you could use the following
Screenshot 2024-03-13 at 23 13 41

You could use the following snippet:
```css
.hubspot-form-wrapper form > div:not(:last-of-type) {
  @apply mb-md;
}
```

- Error message is not correctly aligned: 
Screenshot 2024-03-13 at 23 13 50

File: src/lib/assets/css/form.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+@tailwind utilities;
+
+.hs_name .input input,
+.hs_email .input input,
+.hs_name .input input,
+.hs_company .input input,
+.hs_phone .input input,
+.hs_jobtitle .input input,
+.hs-fieldtype-select .input select {
+  @apply block w-full px-5 py-3 border border-brumosa rounded-m text-grey outline-none focus:rounded-m focus:px-5 focus:py-3 focus:border-teal focus:border-2 focus:border-teal;
```
Comment (mcortes19) [inline src/lib/assets/css/form.css:12]: Remove duplicated `focus:border-teal` class

**Review by mcortes19 — CHANGES_REQUESTED** [review]

**Comment by gstvdesign** [comment]
> ```css
> .hubspot-form-wrapper form > div:not(:last-of-type) {
>   @apply mb-md;
> }
> ```

Thanks for the feedback!

Space added:
Screenshot 2024-03-15 at 10 31 19

Error message aligned properly:
Screenshot 2024-03-15 at 10 31 50

**Review by mcortes19 — APPROVED** [review]
@gstvdesign works like a charm 🎖️

**Pull request merged**